features/step_definitions/searchAmazon.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'selenium-cucumber'
+require 'selenium-webdriver'
+require 'rspec'
+
+Given 'I search {string}' do |value|
+  wait_for_element_to_display(:id, 'twotabsearchtextbox', 10)
+  wait_for_element_to_display(:xpath, '//input[@value="Go"]', 10)
+  enter_text(:id, value, 'twotabsearchtextbox')
+  click(:xpath, '//input[@value="Go"]')
```

Comment:
You can use the "submit" option here because you are submitting a request.
We use click to click others button types. For example, to click navigating sections buttons.
Also, the "submit" option contributes to the cohesion of the code.